text/0202-events-api.md

```diff
@@ -0,0 +1,227 @@
+# Introducing Events API
+
+
+In this OTEP we introduce an Events API that is based on the OpenTelemetry Log signal. In OpenTelemetry's perspective Log Records and Events are different names for the same concept - however, there is a subtle difference in how they are represented using the underlying data model that is described below. Although every language has APIs for logs, they are not all capable of creating events. We will describe why the existing Logging APIs are not sufficient for the purpose of creating events.  It will then be evident that we will need an API in OpenTelementry for creating events. 
+
+We have an option of adding API for both Logs and Events. However, there is a general consensus that we should not have an API in Otel for creating logs since each language already has multiple logging frameworks. Therefore we restrict the API specification below to Events and call it Events API. For logs, it is recommended that end-users continue to use existing Logging APIs and export the logs into OTLP using the LogEmitter SDK. The Events API will offer a subset of the features of LogEmitter SDK and so it will be backed by LogEmitter SDK and the LogRecord data model.
+
+## Subtle differences between Logs and Events
+In OpenTelemetry's perspective Log Records and Events are different names for the same concept. However, there are subtle differences in how they are represented in the underlying LogRecord data model. Logs have a mandatory severity level as a first-class parameter that events do not have, and events have a mandatory name that logs do not have. Further, logs typically have messages in string form and events have data in the form of key-value pairs. It is due to this that their API interface requirements are slightly different.
+
+## Who requires Events API
+
+Here are a few situations that require recording of Events, there will be more.
+* RUM events (Client-side instrumentation)
+* Recording kubernetes events
+* Recording eBPF events
+* Few other event systems described in [example mappings](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/data-model.md#appendix-a-example-mappings) in the data model.
+
+## Can the current Log API interfaces be used for events?
```

Comment:
This list reads to me like the issue is not all languages/runtimes having a structured logging API?

If this is the case it might make more sense to instead have the Log spec say that in those cases Otel logging should provide an API, instead of adding an Events API.

Reply:
> If this is the case it might make more sense to instead have the Log spec say that in those cases Otel logging should provide an API, instead of adding an Events API.

@tsloughter I want to leave the otep at proposing the Logs API only for log appenders initially. If the language maintainers feel the need for an end-user API for structured logs in OTel, they can add that incrementally later. Let me know what you think.